### DISCUSSION TITLE
[2.3.10] strelka-backend up and down every 15min intervals

### DISCUSSION THREAD

**Discussion by Masaya-A** [Security-Onion-Solutions/securityonion#2127] [Unsupported Versions] [answered]
The strelka-backend is up and down every 15min intervals. Is this correct behavior?
I guessed following time line what actually happens...

    ==== strelka-backend is dead ====
7:00:00 : highstate starts strelka-backend
7:00:10 : strelka-backend is completely started
    ==== strelka-backend is running ====
7:15:00 : highstate but strelka-backend is running, so nothing happens
7:15:10 : strelka-backend is killed because "time_to_live" is set 900sec (15min) in backend.yaml
    ==== strelka-backend is dead ====
7:30:00 : highstate starts strelka-backend
7:30:10 : strelka-backend is completely started
   ==== strelka-backend is running ====
7:45:00 : highstate... (continuing this routine)

**Comment by TOoSmOotH** [comment, marked as answer]
Strelka backend stops itself after a while if no new files are there. Its part of how strelka works.

**Reply by Masaya-A** [reply]
Well, you mean this is correct behavior.
Then, could you improve so-status because it respond ERROR when I run so-status during strelka-backend dead periods?

**Reply by defensivedepth** [reply]
@Masaya-A I have created an issue to track this, thanks! https://github.com/Security-Onion-Solutions/securityonion/issues/2153
